<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberMem</class>
 <widget class="QWidget" name="NumberMem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_Start">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Demi Cond</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Demi Cond</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Number Memory</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Numbers">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Level">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>175</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>175</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>175</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Demi</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_your_ans">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your answer:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_correct_ans">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Correct answer:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
